src/modules/vtol_att_control/vtol_att_control_params.c

```diff
@@ -143,8 +180,12 @@ PARAM_DEFINE_FLOAT(VT_ARSP_LP_GAIN, 0.3f);
 /**
  * VTOL Type (Tailsitter=0, Tiltrotor=1, Standard=2)
  *
+ * @unit enum
+ * @value 0 Tailsitter 1 Tiltrotor 2 Standard
```

Comment:
Each value entry has to be on one line. You can't have multiple on a single line. This is structured markup.